editor.planx.uk/src/@planx/components/shared/Preview/SummaryList.tsx

```diff
@@ -639,14 +641,19 @@ function Page(props: ComponentProps) {
     return answer;
   };
 
+  // Omit un-answered optional fields from Review component display
+  const fieldsWithAnswers = fields.filter(
+    (field) => displayValue(answers[0][field.data.fn]) !== "",
+  );
+
   return (
     <>
       <Box component="dt">{props.node.data.title}</Box>
       <Box
         component="dd"
         sx={{ gap: 2, display: "flex", flexDirection: "column" }}
       >
-        {fields.map((field) => (
+        {fieldsWithAnswers.map((field) => (
```

Comment:
Also included a quick fix to how Pages display on "Review" component here: 
- Checklist answers should be comma-separated (still a bit imperfect that Review displays data value rather than user-facing text - let's come back to that one separately!)
- Optional questions that were not answered should be omitted

**Before:**
![Screenshot from 2025-04-29 17-34-51](https://github.com/user-attachments/assets/38ccf32a-dce4-4e54-b1fb-edc2212d8f7b)

**After:**
![Screenshot from 2025-04-29 17-35-14](https://github.com/user-attachments/assets/464b4968-19c1-4e36-8c9a-719ca3314fcf)